### PR TITLE
Fixes styling for top panel to be responsive for different browser sizes

### PR DESCRIPTION
**Pull request by ajainarayanan**

### CODE REVIEW

File: cdap-ui/app/features/hydratorplusplus/toppanel.less [hunk 1]
```diff
@@ -431,6 +440,8 @@ body.theme-cdap {
       .pipeline-specific-section {
         justify-content: flex-end;
         border-left: 0;
+        display: flex;
```
Comment (astewes) [inline cdap-ui/app/features/hydratorplusplus/toppanel.less:443]: I would use the .flexbox() mixin here.

**Comment by astewes** [comment]
One small nit, otherwise LGTM 👍

File: cdap-ui/app/features/hydratorplusplus/toppanel.less (hunk 1 above)
Comment (elfenheart) [inline cdap-ui/app/features/hydratorplusplus/toppanel.less:443]: not necessary with autoprefixer

**Comment by elfenheart** [comment]
👍  LGTM

**Pull request merged**